akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -39,7 +39,7 @@ Java
 
 The first important thing to notice is the `Behavior` of a persistent actor is typed to the type of the `Command`
 because this is the type of message a persistent actor should receive. In Akka Typed this is now enforced by the type system.
-The event and state are only used internally.
+The event and state are only used internally in the `EventSourcedBehavior`.
```

Comment:
Maybe that sentence is just adding confusion. Events can be consumed by read-side. State may be sent as a message if one like (although I wouldn't recommend it).

Reply:
Yeah, I was on the fence. Let's remove it.

Reply:
+1